### ISSUE TITLE
Dockle  0.3.14 fails when image is referenced with `:latest` tag

### ISSUE THREAD

**Issue by marinpurgar** [goodwithtech/dockle#122]
**Description**

Executing the `dockle -i DKL-DI-0006 --exit-code 1 --exit-level warn $ECR_URI:latest` fails with the error `FATAL   unable to initialize a image struct: invalid image name: invalid reference format`.

ECR_URI is the URI of the private ERC Repository with authenticated docker login.

When image is tagged and referenced by any other tag except `:latest` (e.g. the git SHA) dockle executes properly.

**What did you expect to happen?**

Dockle executes normally without failing with non-descriptive error.

**What happened instead?**

Dockle failed with non-descriptive error `FATAL   unable to initialize a image struct: invalid image name: invalid reference format`.

**Output of run with `-debug`:**

```
$ dockle -debug -i DKL-DI-0006 --exit-code 1 --exit-level warn $ECR_URI:latest
DEBUG	There is no .dockleignore file
DEBUG	Fetch latest version from github
DEBUG	Start assessments...
FATAL	unable to initialize a image struct:
  github.com/goodwithtech/deckoder/extractor/docker.newDockerExtractor
        /home/runner/go/pkg/mod/github.com/goodwithtech/deckoder@v0.0.0-20200308110342- e44aa2bd81b3/extractor/docker/docker.go:73
  - invalid image name:
     github.com/goodwithtech/deckoder/extractor/image.NewImage
         /home/runner/go/pkg/mod/github.com/goodwithtech/deckoder@v0.0.0-20200308110342-e44aa2bd81b3/extractor/image/image.go:60
  - invalid reference format
```

**Output of `dockle -v`:**

```
dockle version 0.3.14
```

**Additional details (base image name, container registry info...):**

N/A

**Comment by marinpurgar** [comment]
To demonstrate this issue in simplistic way this is the something one can reproduce very easy:
```$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock goodwithtech/dockle:v0.3.14 goodwithtech/dockle:latest
2021-06-09T13:50:53.619Z	FATAL	unable to initialize a image struct: invalid image name: invalid reference format
(.venv) pmc@desktop:~/Projects/AE/git$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock goodwithtech/dockle:v0.3.14 goodwithtech/dockle:v0.3.14
WARN	- CIS-DI-0001: Create a user for the container
	* Last user should not be root
INFO	- CIS-DI-0005: Enable Content trust for Docker
	* export DOCKER_CONTENT_TRUST=1 before docker pull/build
INFO	- CIS-DI-0006: Add HEALTHCHECK instruction to the container image
	* not found HEALTHCHECK statement
$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock goodwithtech/dockle:v0.3.14 goodwithtech/dockle:latest
2021-06-09T13:51:19.245Z	FATAL	unable to initialize a image struct: invalid image name: invalid reference format
```

**Comment by tomoyamachi** [comment]
@marinpurgar Thank you for reporting.
I fixed it on v0.3.15.
